test/error.spec.ts

```diff
@@ -71,8 +72,26 @@ describe('Error', () => {
           stack: ee.stack,
         },
       ],
+      imtInternalError: {
+        name: 'TypeError',
+        message: 'Type message.',
+        stack: ee.stack,
+      },
+      imtExceptionHash: 'im-hash',
```

Comment:
Not super happy about repeating this one. I was thinking about not serializing it and keeping just the hash, but I guess something in datadog could depend on it for projects that assign this value.